### PR TITLE
Fix decoding byte array

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

File: packages/as-proto-gen/src/generate/field.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ export function generateFieldDecodeInstruction(
     if (isRepeated) {
       return `
         case ${fieldNumber}:
-          if ((tag & 7) === 2) {
+          if ((tag & 7) === 2 && tag !== 26) {
```
Comment (sosedoff) [inline packages/as-proto-gen/src/generate/field.ts:108]: Where is the value `26` coming from?

**Review by sosedoff — APPROVED** [review]
LGTM assuming the value is correct

File: packages/as-proto-gen/src/generate/field.ts (hunk 1 above)
Comment (patiee) [reply packages/as-proto-gen/src/generate/field.ts:108]: 26 is array bytes

Comment (sosedoff) [reply packages/as-proto-gen/src/generate/field.ts:108]: But where is the actual list of these numbers?

**Pull request merged**